snowplow_tracker/payload.py

```diff
@@ -78,15 +78,15 @@ def add_json(self, dict_, encode_base64, type_when_encoded, type_when_not_encode
         """
 
         if dict_ is not None and dict_ != {}:
-
             json_dict = json.dumps(dict_, ensure_ascii=False)
-
             if encode_base64:
-                encoded_dict = base64.urlsafe_b64encode(json_dict.encode("ascii"))
```

Comment:
This PR doesn't solve the issue https://github.com/snowplow/snowplow-python-tracker/issues/194 in my local tests

Simply changing the encoding on this line to `utf-8` does the trick